pages/base.page.js

```diff
@@ -0,0 +1,45 @@
+const PAGE_TIMEOUT = 10000
+
+export default class BasePage {
+
+
+   /**
+    * Open url
+    * 
+    */
+   async open(ruta) {
```

Comment:
Parameters should also be in english.

Reply:
changed...
async open(path) {
       await browser.url(path);
   }